components/log/include/esp_log.h

```diff
@@ -61,6 +61,17 @@ void esp_log_level_set(const char* tag, esp_log_level_t level);
  */
 void esp_log_set_vprintf(vprintf_like_t func);
 
+/**
+ * @brief Get function used to current output log handler
+ *
+ * This function can be used to obtain current log handler before redirect 
+ * log output to other destination, with later time return output to the 
+ * previous destination.
```

Comment:
Rather than this sentence, what do you think about something like:

`Returns a function previously set via esp_log_set_vprintf(), or the default log output handler.`
?

Reply:
Yes, this is the best solution to immediately return a pointer to the current function when set a new in esp_log_set_vprintf ()

Reply:
Returns a function previously set